sites/fast-frame/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "fast-frame",
```

Comment:
This should likely be private and named as the website - not fast frame.

Reply:
just website?